src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -555,7 +555,7 @@ def makeDagSpecs(self, task, sitead, siteinfo, jobgroup, block, availablesites,
         dagSpecs = []
         i = startjobid
         temp_dest, dest = makeLFNPrefixes(task)
-        if task['tm_publication'] :
+	if task['tm_publication'] == 'T' :
```

Comment:
Could you fix two things here? There is a space missing at the start of the line (wrong indentation), as well as an unnecessary space before the colon at the end of the line. Otherwise I'll merge and make the change in an hour so that we can deploy it.